misc/frameworks/frameworks.md

```diff
@@ -101,14 +106,13 @@ small footprint.
 * [Kraken](http://kraken-php.com) - Distributed and async PHP framework.
 * [Lumen](http://lumen.laravel.com/) - Micro-framework by Laravel.
 * [MicroMVC](https://github.com/Xeoncross/micromvc)
-* [Phlyty](https://github.com/phly) - Micro-framewok written using ZF2
-  components.
+* [Phlyty](https://github.com/phly) - Micro-framewok written using ZF2 components.
 * [Proton](https://github.com/alexbilbie/Proton) -
-  [StackPHP](http://stackphp.com/)-compatible micro-framework.
+* [StackPHP](http://stackphp.com/) - compatible micro-framework.
 * [Slim](http://www.slimframework.com/)
+* [Silex](https://silex.symfony.com/) - The PHP micro-framework based on the Symfony Components
```

Comment:
Silex has been removed in favor of Symfony and Flex. I think we don't need to list it here.

Reply:
Silex is still getting security updates, and its a great start from someone hows getting into frameworks, the documentation is still up too,